main.go

```diff
@@ -210,4 +212,11 @@ func initFlags(fs *pflag.FlagSet) {
 		10*time.Minute,
 		"The minimum interval at which watched resources are reconciled.",
 	)
+
+	fs.StringVar(
+		&util.ProviderIDFormat,
+		"providerid-format",
+		"v2",
```

Comment:
wondering if we can set `v1` as default

Reply:
It is supposed to be v1, I think i missed while testing, Will set back to v1